androgee/androgee.py

```diff
@@ -20,8 +22,15 @@ async def spray(ctx, member: discord.Member = None):
     if member == None:
         await ctx.send(file=image)
     else:
-        message = f"{member.mention} was sprayed by {ctx.message.author.mention}"
-        await ctx.send(message, file=image)
+        if mod_role_id in [y.id for y in ctx.author.roles]:
+            await member.edit(mute=True)
+            message = f"{member.mention} was sprayed by {ctx.message.author.mention}"
+            await ctx.send(message, file=image)
+            await asyncio.sleep(10)
```

Comment:
Mee6 already has a great command called tempmute. We could have Androgee tell Winston to temporarily mute the person in question and provide a reason.

`!tempmute [member] [duration] (optional reason)`

So like so:

`ctx.send(f"!tempmute {member.mention} 30 being too rowdy")`

or something like that